examples/dependencies/up.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+docker-compose up -d
+
+# Initialise the Rabbitmq instance
+sleep 15
+printf "\n\nInitialising the Rabbimq instance...\n\n"
+./rabbitmq_setup.sh
+
+# Initialise the schemas
+printf "\n\nInitialising the Redpanda instance...\n\n"
+./schemas/push_schemas.sh http://localhost:8081 secret-key
```

Comment:
Not sure if you're bothered, but generally all scripts should end with a new line character.  Sometimes it matters, sometimes it doesn't, so I just put them in every time unless the language in the file specifically doesn't want one.

Reply:
Ok, I didnt know